qa/rpc-tests/sc_ft_and_mbtr_fees_update.py

```diff
@@ -23,13 +22,6 @@
 
 NUM_TXS       = 10
 
-#### temporary logs functions
-def cc(c, inp):
-    return inp
-def mark_logs_temp(msg, nodes, debug = 0, color = 'n'):
-    return mark_logs(msg, nodes, debug)
-####
-
 class provaFee(BitcoinTestFramework):
```

Comment:
Just seen this accidentally, probably we want to take this chance to rename `provaFee` class (being the file under refactoring).